src/main.rs

```diff
@@ -37,16 +37,17 @@ async fn graphiql() -> impl IntoResponse {
 /// Establishes database connection and returns the client.
 async fn db_connection() -> Client {
     // Parse a connection string into an options struct.
-    let mut client_options = ClientOptions::parse("mongodb://db:27017").await.unwrap();
+    let mut client_options = ClientOptions::parse("mongodb://wishlist-db:27017").await.unwrap();
```

Comment:
imho it would be good to make this configurable via environment variables
also, how about credentials for the db?

Reply:
MongoDB is never exposed to the host, should we really make its port an environment variable?

Reply:
not only the port, but the whole connection string
in the docker compose setup this might be the correct address
but what if for example someone wants to test a shared database instance (for whatever experiment reasons)?
then this would change